src/events/crawler/scrapers/RUS/index.js

```diff
@@ -0,0 +1,39 @@
+import needle from 'needle';
+
+const scraper = {
+  country: 'RUS',
+  aggregate: 'state',
+  url: 'https://yandex.ru/maps/api/covid?csrfToken=',
```

Comment:
Can you include a `curators: [{name: 'Yandex'}]` etc bit to cite Yandex as the curator?

Also, if Yandex cites its sources, can you include `sources: [{name: 'etc', url: 'etc'}]`?

Reply:
How's this?